CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.0.0-beta2 (unreleased)
+
+* [Breaking] Remove `ulog` and expose `*zap.Logger` directly.
+* [Breaking] Upgrade `zap` to `v1.0.0-rc.2` (now go.uber.org/zap, was
+    github.com/uber-go/zap)
```

Comment:
We should be able to migrate most code with `gofmt -r`. Do you normally provide migration scripts to help users?

(Also note that I didn't update the `ulog` docs - someone in UberFX may want to take care of that before the next release.)